docker-compose.yml

```diff
@@ -8,20 +8,41 @@ services:
       - POSTGRES_USER=${DB_USER}
 
   court_server:
-    build: ./packages/server
+    build: .
     image: aragon/court-server
+    command: npm run start:server:dev
     depends_on:
       - court_server_db
     ports:
-      - "8080:8080"
-      - "8081:8081"
+      - "8000:8000"
+    volumes:
+      - ./packages/server/src:/court-backend/packages/server/src
     env_file:
       - .env
 
   court_services:
-    build: ./packages/services
+    build: .
     image: aragon/court-services
+    command: npm run start:services:dev
     depends_on:
       - court_server
+    volumes:
+      - ./packages/services/bin:/court-backend/packages/services/bin
+      - ./packages/services/src:/court-backend/packages/services/src
+    env_file:
+      - .env
+
+  court_backoffice:
+    build: .
+    image: aragon/court-backoffice
```

Comment:
Just wanted to say that in this case you are not creating another image, just adding a tag to the same image as before.
So you end up with the same Dockerfile image with 3 tags:
- aragon/court-server, services and backoffice.
I would just call this court-backend in all 3 cases to avoid confusion.
(The only difference is in how you execute it, i.e. the `command` line